<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePageWidget</class>
 <widget class="QWidget" name="TemplatePageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>613</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="mainHorizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="templateLabel">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>390</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>390</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>拖动右侧模板到此以更换相册背景</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>相册模板</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="sideFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="innerVerticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>版式：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QCheckBox" name="checkBox_13">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>田野</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>颜色：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkBox_15">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="checkBox_11">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>卡通</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QCheckBox" name="checkBox_21">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1张</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>2张-横幅×2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCheckBox" name="checkBox_19">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>黄色</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QCheckBox" name="checkBox_23">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>3张</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6" colspan="2">
         <widget class="QCheckBox" name="checkBox_26">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>5张</string>
          </property>
         </widget>
        </item>
        <item row="4" column="8">
         <widget class="QCheckBox" name="checkBox_27">
          <property name="text">
           <string>大于5张</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6" colspan="2">
         <widget class="QCheckBox" name="checkBox_14">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>秋天</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>风格：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>儿童</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QCheckBox" name="immediatelyCheckBox">
          <property name="text">
           <string>开启即刻搜索</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBox_10">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>古典</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QCheckBox" name="checkBox_12">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>海军</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox_16">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>蓝色</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_9">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="checkBox_17">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>绿色</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6" colspan="3">
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>3张-小横幅×3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>写真</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>1张-竖幅×1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QCheckBox" name="checkBox_18">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>红色</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="checkBox_22">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>2张</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="checkBox_20">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全部</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QCheckBox" name="checkBox_25">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>4张</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>婚纱</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>48</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>种类：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>照片数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="9">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="resetPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="searchPushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="templatesGraphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>448</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="renderHints">
         <set>QPainter::TextAntialiasing</set>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
